src/app/angular2-multicomponent-form.component.ts

```diff
@@ -22,13 +21,35 @@ import { SidebarComponent } from './sidebar/sidebar.component';
 ])
 export class Angular2MulticomponentFormAppComponent {
   title = 'Angular2 Multi Component Dynamic Form demo';
+  baseHref = '/angular2-multicomponent-form'; 
```

Comment:
this is not for base href setting. though base href is been set in index.html directly. i wanted that becoz since in IE9 on default route url been changed to `/#/base/route` and i was appending the base url for matching with route path..